### PR TITLE
Implement docker build/push in CICD

### LINKED ISSUE #6: Create the docker image and push it to docker hub as part of the CICD
## 💡 Description

As what is done for the other registry components (registry-loader, registry-api for example).

### PR DESCRIPTION
**Pull request by alexdunnjpl**
## ⚙️ Test Data and/or Report
Manually tested

## ♻️ Related Issues
fixes #6 

@jordanpadams @jimmie N.B. the docker tag version for releases is pulled directly from the python module version number, located at [./src/pds/registrysweepers/VERSION.txt](https://github.com/NASA-PDS/registry-sweepers/blob/main/src/pds/registrysweepers/VERSION.txt).

If this is not updated correctly during the release process, old versions of the images on dockerhub will be overwritten.  This seems brittle, so if we have a better way of doing this, let me know.

### CODE REVIEW

File: .github/workflows/stable-cicd.yaml [hunk 1]
```diff
@@ -73,6 +73,31 @@ jobs:
                     pypi_username: ${{secrets.PYPI_USERNAME}}
                     pypi_password: ${{secrets.PYPI_PASSWORD}}
                     ADMIN_GITHUB_TOKEN: ${{secrets.ADMIN_GITHUB_TOKEN}}
+            -
+                name: 🫙 Docker Image Tag Determination
+                id: docker_tag_determination
+                run: echo "::set-output name=image_tag::$(cat ./src/pds/registrysweepers/VERSION.txt)"
```
Comment (nutjob4life) [inline .github/workflows/stable-cicd.yaml:79]: FYI, we'll need to move `::set-output` → `$GITHUB_OUTPUT` in the near future

File: .github/workflows/stable-cicd.yaml [hunk 2]
```diff
@@ -73,6 +73,31 @@ jobs:
...
+                name: 🎰 QEMU Multiple Machine Emulation
+                uses: docker/setup-qemu-action@v2
+            -
+                name: 🚢 Docker Buildx
+                uses: docker/setup-buildx-action@v2
+            -
+                name: 🧱 Image Construction and Publication
+                uses: docker/build-push-action@v3
+                with:
+                  context: ./
```
Comment (nutjob4life) [inline .github/workflows/stable-cicd.yaml:96]: ↑ 4 spaces
↓ 2 spaces

Not a deal-breaker, but whatevs 🤷

**Review by nutjob4life — APPROVED** [review]
👍

**Comment by alexdunnjpl** [comment]
Converting to draft while I work out a bug I just noticed - the docker-hub image is tagged 1.2.0 despite the VERSION.txt content being 1.1.0

File: .github/workflows/stable-cicd.yaml (hunk 2 above)
Comment (alexdunnjpl) [reply .github/workflows/stable-cicd.yaml]: fixed

File: .github/workflows/stable-cicd.yaml (hunk 1 above)
Comment (alexdunnjpl) [reply .github/workflows/stable-cicd.yaml:67]: Got a reminder for this and at least one other repository - need to enumerate affected repos

**Pull request merged**